hass_nabucasa/cloudhooks.py

```diff
@@ -41,6 +41,7 @@ async def async_create(self, webhook_id: str, managed: bool) -> Dict[str, Any]:
         with async_timeout.timeout(10):
             resp = await cloud_api.async_create_cloudhook(self.cloud)
 
+        resp.raise_for_status()
```

Comment:
I think we should raise a own Exception, catch that on HA and make a better error handling with a nice message. Now we replace the exception from json with the exception from aiohttp.

Reply:
Let's do that in another PR. For now we should at least raise. Currently you get a key error because it treats it as valid data.